packages/api/src/derives/index.ts

```diff
@@ -20,10 +20,11 @@ import {AnyFunction} from '@cennznet/types/types';
 import {ApiInterfaceRx, MethodResult} from '@polkadot/api/types';
 import {Observable} from 'rxjs';
 import * as fees from './fees';
+import * as cennznetStaking from './staking';
 
 export type DeriveFunc = (api: ApiInterfaceRx) => (...args: any[]) => Observable<any>;
 
-export const derive = {attestation, cennzxSpot, fees, genericAsset};
+export const derive = {attestation, cennzxSpot, fees, cennznetStaking, genericAsset};
```

Comment:
Is there anyway to augment api.derive.staking or avoid calling this `cennznetStaking`? this is the cennznet API so it's implied

Reply:
Since polkadot already supports staking derive queries at node_modules/@polkadot/api-derive/staking
So I was afraid to use staking cause i felt it will override their changes and we might not able to use them..

Reply:
can you please suggest some good name over cennznetStaking?

Reply:
I though we are implementing this because the polkadot version is not working? if so we should override it?